src/FrameHelper.cpp

```diff
@@ -133,8 +133,9 @@ namespace frame_helper
             base::samples::frame::Frame &dst,
             int offset_x,
             int offset_y,
-            ResizeAlgorithm algo,
-            bool bdewrap)
+            ResizeAlgorithm const& resize_algorithm,
+            bool bdewrap,
+            UndistortAlgorithm const& undistort_algorithm)
```

Comment:
ResizeAlgorithm and UndistortAlgorithm are both enum. It's the one case where arguments should be passed by value.